src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO Logo Header"
+        />
+      </a>
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-list-link-item">
+            <a
+              href="/"
+              class="nav-list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-list-link-item">
```

Comment:
<img width="1062" alt="Знімок екрана 2024-08-13 о 15 36 37" src="https://github.com/user-attachments/assets/58fc139c-6773-4be0-a434-aff6aa878bcf">

```suggestion
          </li>
          
          <li class="nav-list-link-item">
```

In all the cases!